lib/llvm/llvm-utils.h

```diff
@@ -1,11 +1,13 @@
 #ifndef DG_LLVM_UTILS_H_
 #define DG_LLVM_UTILS_H_
 
+#include <llvm/ADT/iterator_range.h>
 #include <llvm/IR/DataLayout.h>
 #include <llvm/IR/Function.h>
 #include <llvm/IR/Instructions.h>
 #include <llvm/IR/IntrinsicInst.h>
 #include <llvm/IR/Type.h>
+#include <llvm/Support/raw_ostream.h>
```

Comment:
Why?

Reply:
That's a good question. Honestly, I don't know,.

Reply:
And without this include the code does not work?

Reply:
It does work. It was probably committed by an accident. Dropped in 96f2ebf6.

Reply:
Alright, even though it did work on my machine, it fails in the Docker job:
```
FAILED: lib/CMakeFiles/dgllvmvra.dir/llvm/ValueRelations/StructureAnalyzer.cpp.o 
/usr/lib/ccache/g++ -DENABLE_CFG -DHAVE_TSL_HOPSCOTCH -DUSING_SANITIZERS -Ddgllvmvra_EXPORTS -I/home/runner/work/dg/dg/include -I/home/runner/work/dg/dg/lib -isystem /usr/lib/llvm-10/include -isystem /home/runner/work/dg/dg/tsl-hopscotch/include -fdiagnostics-color=always -std=c++14 -fno-rtti -Wall -Wextra -fno-exceptions -fsanitize=undefined,address -g -fno-omit-frame-pointer -fno-sanitize-recover=all -O2 -g -DNDEBUG -fPIC   -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -std=c++14 -MD -MT lib/CMakeFiles/dgllvmvra.dir/llvm/ValueRelations/StructureAnalyzer.cpp.o -MF lib/CMakeFiles/dgllvmvra.dir/llvm/ValueRelations/StructureAnalyzer.cpp.o.d -o lib/CMakeFiles/dgllvmvra.dir/llvm/ValueRelations/StructureAnalyzer.cpp.o -c /home/runner/work/dg/dg/lib/llvm/ValueRelations/StructureAnalyzer.cpp
In file included from ../lib/llvm/ValueRelations/StructureAnalyzer.cpp:3:
../lib/llvm/llvm-utils.h: In function ‘void dg::llvmutils::printerr(const char*, const llvm::Value*, bool)’:
../lib/llvm/llvm-utils.h:69:16: error: ‘errs’ was not declared in this scope
   69 |     print(val, errs(), msg, newline);
      |                ^~~~
```

Reply:
Ok, then it is needed. But maybe would be nice to have it in its own commit as this is not related to the commit it is currently in.

Reply:
Done in 18908da.